### PR TITLE
ccy_translate function added

### PR DESCRIPTION
**Pull request by btrdnch**

### CODE REVIEW

File: test/index.test.js [hunk 1]
```diff
@@ -1,66 +1,118 @@
 'use strict'
 
-const chai = require('chai').use(require('dirty-chai'))
-const { expect } = chai
+const mockRestConf = jest.fn()
```
Comment (vigan-abd) [inline test/index.test.js:3]: usually we use mocha as standard lib for tests and not jest, please restore mocha and use https://www.npmjs.com/package/bfx-api-mock-srv as mock lib for restv2 test

**Review by vigan-abd — CHANGES_REQUESTED** [review]

File: test/index.test.js (hunk 1 above)
Comment (btrdnch) [reply test/index.test.js:3]: hmm, i see, alright

Comment (btrdnch) [reply test/index.test.js:3]: @vigan-abd there is no `pub:map:currency:sym` method support [here](https://github.com/bitfinexcom/bfx-api-mock-srv/blob/master/lib/rest2/methods.js#L10). Should I add it to the lib or should I just mock the call in place?

Comment (btrdnch) [reply test/index.test.js:3]: as discussed, there is 1 more thing: the mock api works on the localhost while the lib uses pub bitfinex api and there is no possibility to switch api url

**Review by vigan-abd — APPROVED** [review]
lgtm

**Comment by prdn** [comment]
the purpose of lib-js-util-* is to be nice / self contained
it should not do REST calls to endpoints to get additional conf

**Pull request closed without merging**